centreon/www/install/createTables.sql

```diff
@@ -10,92 +10,92 @@
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
+/*!40101 SET SET character_set_client = utf8mb4 */;
```

Comment:
You have a double SET in the whole file:
```suggestion
/*!40101 SET character_set_client = utf8mb4 */;
```